### ISSUE TITLE
Couldn't you have told us you're invite only before registering and confirming email?

### ISSUE THREAD

**Issue by miro-ux** [black-forest-labs/flux#38]
![image](https://github.com/user-attachments/assets/42ab9b1d-f5e8-4d60-b93a-520bdb450650)

🤷🏻‍♂️

...or am I doing it wrong? Should we use replicate to access it?

**Comment by JonasLoos** [comment]
The first thing, when clicking "Use FLUX.1 [pro] on our API", is a disclaimer that their API is currently only open for "selected partners" ([source](https://docs.bfl.ml/)):

![image](https://github.com/user-attachments/assets/1347b312-4307-48ad-ba7b-632e2848a7ff)

On replicate you can use it directly, so yes, this is what I would suggest to you.

**Comment by miro-ux** [comment]
@JonasLoos Oops my bad then... 
But regardless IMHO, it's not clear enough that it's invite-only and what "select partners" means and also why allow easy access to registrations in that case? Just my 2 cents... I think it's bad UX for sure and I felt tricked into registring.

**Comment by JonasLoos** [comment]
@miro-ux yes, I was also somewhat confused when reading this for the first time

**Comment by ascheucher-shopify-partner** [comment]
It's not just bad UX, it's a dark UX pattern to collect emails to contact later on.

Really leaves a bad taste.
